src/main/haskell/language-kore/src/Data/Kore/ASTPrettyPrint.hs

```diff
@@ -133,9 +134,9 @@ writeFieldNewLine fieldName field object = do
         (betweenLines >> prettyPrint MaySkipParentheses (field object))
 
 writeAttributesField
-    :: (PrinterOutput w m, PrettyPrint (Fix (pat variable)))
+    :: (PrinterOutput w m, PrettyPrint (Attributes))
```

Comment:
here you shouldn't need Attributes, as an instance for it should already be satisfied.